### PR TITLE
Adds ImageBinarization to Images.jl

### PR DESCRIPTION
**Pull request by ashwani-rathee**
- [x] Adds ImageBinarization to Images.jl

### CODE REVIEW

File: .github/workflows/UnitTest.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        julia-version: ['1.3', '1', 'nightly']
+        julia-version: ['1.6', '1', 'nightly']
```
Comment (johnnychen94) [inline .github/workflows/UnitTest.yml:17]: If we drop the CI for Julia < 1.6, then it's better to change `julia = "1.3"` in the `[compat]` section in Project.toml as well

**Review by johnnychen94 — APPROVED** [review]

**Comment by johnnychen94** [comment]
I'm okay to have ImageBinarization in Images, but since I'm not working actively here, it's better for others to merge.

File: .github/workflows/UnitTest.yml (hunk 1 above)
Comment (ashwani-rathee) [reply .github/workflows/UnitTest.yml:17]: Okay 👍🏻 ,Is it alright that I am changing version to 0.26.0, assuming it's kind of breaking?

Comment (johnnychen94) [reply .github/workflows/UnitTest.yml:17]: If so, it might be a good idea to merge #1011 and perhaps #1027 for 0.26

Comment (ashwani-rathee) [reply .github/workflows/UnitTest.yml:17]: So I merge this first and then see what could be done about #1011 and #1027? After all that is done, then release?

Comment (ashwani-rathee) [reply .github/workflows/UnitTest.yml:17]: Also I again don't have merge rights here and for good reasons 🤣

**Pull request merged**